lib/config_cat.ex

```diff
@@ -134,6 +134,13 @@ defmodule ConfigCat do
     ConfigCat.get_value("setting", "default", client: :unique_name)
     ```
 
+  - `offline`: **OPTIONAL** timeout for receiving an HTTP response from
+    the socket, in milliseconds. Default is 5000.
+
+    ```elixir
+    {ConfigCat, [sdk_key: "YOUR SDK KEY", read_timeout_milliseconds: 5000]}
+    ```
+
```

Comment:
Looks like this documentation/description is copy-pasted from another option and needs to be updated.